src/Func.h

```diff
@@ -560,6 +560,11 @@ class Func {
                                   const Target &target = get_target_from_environment());
     // @}
 
+    /** Statically compile this function to an object file that is suitable for use
```

Comment:
Explain in detail exactly *how* someone can use the produced object file with matlab.